packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -34,9 +34,15 @@ const selectTargetAccountPrompt = async (accounts, defaultAccountConfig) => {
   let disabledMessage = false;
 
   if (sandboxUsage['DEVELOPER'] && sandboxUsage['DEVELOPER'].available === 0) {
-    disabledMessage = i18n(`${i18nKey}.sandboxLimit`, {
-      limit: sandboxUsage['DEVELOPER'].limit,
-    });
+    if (sandboxAccounts.length < sandboxUsage['DEVELOPER'].limit) {
+      disabledMessage = i18n(`${i18nKey}.sandboxLimitWithSuggestion`, {
+        limit: sandboxUsage['DEVELOPER'].limit,
```

Comment:
There's a new util that we're using whenever we want to log out command references. It's [uiCommandReference](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/ui.js#L89). Can you use that here?

Just a note, it's possible that it won't play nicely with the prompt lib and how that logs. Feel free to keep this as is if uiCommandReference isn't working for it.

Reply:
Nice, looks great with the util usage:
<img width="1202" alt="Screenshot 2023-08-16 at 13 40 26" src="https://github.com/HubSpot/hubspot-cli/assets/16788677/d1d36280-246a-454a-9934-5f3d00c5a7d2">